src/Components/BookBrowser/BookHub.js

```diff
@@ -0,0 +1,222 @@
+import React, { useState, useEffect } from 'react';
+import { PropTypes } from 'prop-types';
+import BookList from '../BookList/BookList';
+import { gradeRangeMetadata, ageRangeMetadata } from '../Filtering/Filtering';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function CardsDisplay({
+  searchTerms, searchCategory, alignment, rangeInput, multiSelectInput,
+}) {
+  const [allBooks, setAllBooks] = useState([]);
+  const [filteredBooks, setFilteredBooks] = useState([]);
+  let incomingGradeIndices = [0, 0];
+  let incomingAgeIndices = [0, 0];
+  const recordAgeIndices = [0, 0];
+  const recordGradeIndices = [0, 0];
+
+  const getCards = () => {
+    base('Book').select({ view: 'Grid view' }).all()
+      .then((records) => {
+        setAllBooks(records);
+      });
+  };
+
+  const isMatchTitleDescIdentity = (book) => {
+    // Filter function for books stored in 'books' state
+    /* FOR FUTURE DEV
+        On Airtable, there is identity_tags and identity. We may need to change this
```

Comment:
this is good as is! identity is the text field that is searchable and identity tags should be filterable